WTWSMS/common/religions/00_religions.txt

```diff
@@ -367,9 +367,11 @@ christian = {
 		can_grant_claim = yes
 		can_call_crusade = no
 		priests_can_inherit = no
-		max_wives = 4
+		max_wives = 1
```

Comment:
There's nothing to substantiate that. Hell, the earliest Irish laws stress that a man has only one wife and that any lovers/mistresses he may have don't enjoy the same rights (such as divorce if he should beat her, be accused of adultery or a homosexual).